api-gateway/src/main/java/com/petclinic/bffapigateway/dtos/Auth/UserDetails.java

```diff
@@ -19,5 +19,6 @@ public class UserDetails {
     private String email;
 
     private Set<Role> roles;
+    private boolean isDisabled;
```

Comment:
I would change the name of this boolean, it's a bit weird, maybe we make it to "isActivated"? or something of sorts. But your logic looks alright, good PR

Reply:
![image](https://github.com/user-attachments/assets/95d30d23-198c-438c-b8df-3912757676e3)
I understand your point, but the naming fits with the code as if you look at my method when you setDisable it becomes true, but your right il change it for next sprint